lib/setup/profile.js

```diff
@@ -8,6 +8,9 @@ var util = require('util');
 var chalk = require('chalk');
 var isNoEntry = require('../util/codes').isNoEntry;
 
+var bashProfileFileName = '.bash_profile';
+var bashrcFileName = '.bashrc';
+
```

Comment:
Given the comment about removing the branching condition, I think these globals can go away.